src/main/java/org/jfrog/hudson/util/PluginDependencyHelper.java

```diff
@@ -22,7 +22,7 @@ public static FilePath getActualDependencyDirectory(File localDependencyFile, Fi
             throws IOException, InterruptedException {
 
         File localDependencyDir = localDependencyFile.getParentFile();
-        String pluginVersion = Hudson.getInstance().getPluginManager().getPlugin("artifactory").getVersion();
+        String pluginVersion = Hudson.get().getPluginManager().getPlugin("artifactory").getVersion();
```

Comment:
If deprecated methods are replaced anyway, it's also suggested to replace `Hudson` with `Jenkins`.